packages/sitecore-jss-nextjs/src/middleware/redirects-middleware.test.ts

```diff
@@ -411,7 +414,8 @@ describe('RedirectsMiddleware', () => {
           status: 301,
           setCookies,
         });
-        const nextRewriteStub = sinon.stub(NextResponse, 'rewrite').callsFake((url) => {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        const nextRewriteStub = sinon.stub(NextResponse, 'rewrite').callsFake((url, _init) => {
```

Comment:
do we need _init here?

Reply:
@art-alexeyenko Yes, we are calling the two parameter version of this method, so matching the call signature for the fake.